src/apis/jsonapi/ApiUtils.ts

```diff
@@ -0,0 +1,75 @@
+import { Serializer, Deserializer } from 'jsonapi-serializer';
+
+export default class ApiUtils {
+  // region public static methods
+  public static async postJsonResource(url: string, serializer: Serializer, resource: any): Promise<any> {
+    const requestOptions = {
+      method: 'POST',
+      headers: ApiUtils.headers(),
+      body: JSON.stringify(serializer.serialize(resource)),
```

Comment:
I'm not sure, if we really need to _stringify_ this again.
I think the output of `serialize` is already stringified.
See [source](https://github.com/fotinakis/jsonapi-serializers/blob/db48bdf1a8d1e2f8559b2c250250310d72517346/lib/jsonapi-serializers/serializer.rb#L276) and the examples at [the docs](https://www.npmjs.com/package/jsonapi-serializer#serialization).

Reply:
`serializer.serialize` sadly returns a javascript object instead of a string, which is why `JSON.stringify` is necessary at this point.